newrelic-api/src/main/java/com/newrelic/api/agent/HttpParameters.java

```diff
@@ -212,6 +251,12 @@ public static UriParameter library(String library) {
 
     public interface Build {
 
+        /**
+         * Set the status code/text of the HTTP response.
+         * @return the builder in a buildable state.
+         */
+        Build status(Integer statusCode, String statusText);
```

Comment:
This is another breaking change. But should only cause problems to someone implementing this interface.
Not sure why one would do it.